.github/workflows/on-pull-request.yml

```diff
@@ -102,3 +102,6 @@ jobs:
       - name: Unit Test
         run: yarn test:ci
 
+      - name: Typecheck Test
```

Comment:
We want to make sure that during the run we test our new function

![image](https://github.com/user-attachments/assets/21b6b896-d0df-4e21-a119-8043819ec1a6)

Reply:
![image](https://github.com/user-attachments/assets/66115d5f-89e3-4eed-a957-39a475132492)